### ISSUE TITLE
I can't use mgit because of multiple errors

### ISSUE THREAD

**Issue by Ninesideddevelopment** [maks/MGit#731]
I've been trying to use mgit on my android device for several days now. I keep getting multiple auth errors when pushing, but now I can't even add repositories. I looked at other issues but they kept having answers like "I used a token" or "use an email instead of a username", both of which I tried but it failed.

I'd appreciate this a lot if some could help with this, however I have noticed answers are usually referrals to other issues, which either have more referrals, or solutions that don't work for me. I don't mean to be difficult and I'm sorry if I am being so, but I would ideally like the answers to stay in this issue/thread. Thank you :)

**Comment by maks** [comment]
I'm sorry but I can't really help with more details regarding what specific error you are getting? did pushing to that specific Github repo previously work and if so when did it stop working? how are you accessing the remote, https or ssh? what type of credentials are you using? is you are using email and token, are you using a Personal access token? if so are you using a "classic" or fine grained token? 

If you could provide more details for the above, I can try to assist you further.

**Comment by Ninesideddevelopment** [comment]
For credentials, I tried username and password first, then email and password, then I tried email and classic personal access token, then email and a fine grained personal access token.

I first tried https, then ssh, neither of which worked. 

I was getting errors such as, "authentication error: <reponame>: not authorised", and an error that just said "authentication error".

I think it could just be my phone because I reinstalled the app and and it's working OK now. Sorry I hadn't given you this information prior.
